opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -44,6 +44,33 @@ message InstrumentationLibraryMetrics {
   repeated Metric metrics = 2;
 }
 
+// A representation of raw measurements, which can have statistical meaning based
+// on how the measurement was sampled, as well as a span ID and trace ID of the
+// active span when the measurement was recorded
+message RawValue {
+  // The set of labels that were dropped by the aggregator, but recorded
+  // alongside the original measurement. Only labels that were dropped by the aggregator should be included
+  repeated opentelemetry.proto.common.v1.StringKeyValue labels = 1;
+
+  // Exact time that the measurement was recorded
+  fixed64 time_unix_nano = 2;
+
+  // Numerical value of the measurement that was recorded. Only one of these 
+  // two fields is used for the data, based on MetricDescriptor.measurement_value_type
+  double double_value = 3;
+  int64 int64_value = 4;
+
+  // (Optional) Span ID of the current trace
+  bytes span_id = 5;
```

Comment:
Would be good to explain that this may be missing if the measurement is not recorded inside a "request"/"trace"

Reply:
added comments